### ISSUE TITLE
No option to review

### ISSUE THREAD

**Issue by erosman** [mozilla/olympia#1556]
There is something wrong with Olympia... it doesn't give me the option to review, only "Request more information Request super-review Comment" !!??

eg:
PerfectPixel 1.0.10 Extension **1 day**
Brief 2.2.1 Extension **21 hours**
Facebook Share Button 44.0 Extension **21 hours**
MPL Koha Patch 16.4.0 Extension **20 hours**

**Comment by jvillalobos** [comment]
This should only happen the most recent submissions (under 20 hours, I think), which shouldn't be visible on the queues. The ones that appear in the queues should be reviewable. Please reopen if this is happening under other circumstances.

**Comment by erosman** [comment]
As you can see from the post......
- All have already appeared on the queues
- Some says 20 hours and some even 1 day
- A couple of 1 day old ones were fine to review few hours ago but it changed now (when I posted)

PS. Not the issue here but entries used to start form 7 hours ago and now it says 20 hours ago. Did they really upload them 20 hours ago?

**Comment by erosman** [comment]
They now say 23 hours, 1 day etc and the situation is still the same. (unless it is only me having this issue ie I am being blocked!!??)
There is something wrong and the issue must be opened.
I am not able to re-open an issue closed by repo collaborators.

**Comment by jvillalobos** [comment]
Okay, I'm reopening so someone can look at these cases. They should only be blocked when submitted in the last 20 hours.

**Comment by erosman** [comment]
I can not review any of these:

Brief 2.2.1  Extension  1 day
Facebook Share Button 44.0  Extension  1 day
MPL Koha Patch 16.4.0  Extension  1 day
Twitter App 44.0 Extension  1 day
YouTube Video Replay 44.0 Extension  1 day
OPIE (Ordered Preference Import/Export) 6.4 Extension  21 hours
Puzzle Bars 2.1.9 Extension  21 hours
Stylish Editor Modified 47.0.1 Extension  21 hours

SalesforceOverlay_Beta 1.1.34 Extension   1 day
telejurnal.com 1.1.1 Extension  1 day
OPIE2 1.0.6 Extension   21 hours

Check My HTTPS 2.0.1  Extension   21 hours

**Comment by erosman** [comment]
I have just noticed something .... 
Is Exclusive  an admin reviewer?
He was able to review 2 addons that I was unable to review (ie the review options did not appear)
```
https://addons.mozilla.org/en-US/editors/review/firegestures
https://addons.mozilla.org/en-US/editors/review/facebook-share-button
```

If he is not an admin reviewer, is it possible that someone coded something using local time (instead of a fixed reference eg GMT)?

**Comment by erosman** [comment]
Just a note to say that the situation has not changed (although I realise that it has been the weekend).

The above bug, combined with addon-hoarding (where all new addons are opened and locked for 12+ hours until they become available to review) has considerably hampered the review process.

What was the logic behind the new changes; to limit the reviewers ability to review; to prevent new addons from being reviewed quickly and forcefully make new uploads wait at least 24+ hours; only make new uploads available to the Admin reviewers?

What is the point of addons that can not be reviewed appearing on the queue (which are annoying reviewers who open them to find out they can not be reviewed)?

**Comment by jvillalobos** [comment]
The purpose is to give less active reviewers some add-ons to review, while allowing the most active ones to still handle most of them. Andreas is working on fixing this bug and looks like the fix will go live next week due to the release schedule. On a positive note, Andreas's fix will also fix the problem where add-ons appear in the queues with 8 hours waiting time. For now, I suggest you don't review any add-ons that show up in the queue with less than one day waiting time.

**Comment by erosman** [comment]
> The purpose is to give less active reviewers some add-ons to review, while allowing the most active ones to still handle most of them.

Is that to say that for example, I have been personally targeted and barred from reviewing newly uploaded addons?
In other words, an institutional discriminatory system has been created to intentionally target the most reliable and consistent volunteer reviewers who have been donating their time and effort for free for months and years, on weekdays, weekends, and on holidays while everyone else was away?!!!

> For now, I suggest you don't review any add-ons that show up in the queue with less than one day waiting time.
I am not able to review addons that have been waiting in the queue for less than a day, whether I want to or not.

With the additional situation of addon-Hoarding (sometimes for 24+ hours, check the image in the previous post), addons that have been around for over 1 day are not available either.

Well... as you wish

**Comment by jvillalobos** [comment]
At the moment Teo and you are in the group that can't review recent submissions, but we'll update it group as needed. We would have preferred to rely on more of an honor code for this, but clearly that wasn't going to work after discussing it with you.

Your work is greatly appreciated, and the impact to Teo and you should be minimal since the amount of reviews taken by less active reviewers is a small fraction of what you two handle.

I don't quite understand the issue about hoarding, but it sounds like something that should be handled by email first so we can figure out what's going on and what should be done. This particular issue is going to be resolved in #1562 and #1594.
